__tests__/implementation.test.js

```diff
@@ -276,4 +276,89 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles (sorting queries)", () => {
+    it("Should respond 200 ", () => {
+      return request(app)
+        .get("/api/articles")
```

Comment:
I recommend removing this test as you already have other tests that test /api/articles